### ISSUE TITLE
placement attribute is not working.

### ISSUE THREAD

**Issue by narainsagar** [pleerock/ng2-popover#10]
Hi, I am using below popover (from usage guide). but the placement attribute is not working. Can anyone help me. Thanks

```
<button class="info-button"
    [popover]="myPopover">
      Popover Button
</button>

<popover-content #myPopover 
    placement="right">
   Hello World!
</popover-content>
```

**Comment by MikeAirey** [comment]
I had the same issue. Putting the placement attribute on the button worked for me:
`<button class="info-button"
    [popover]="myPopover"
    popoverPlacement="right">
      Popover Button
</button>

<popover-content #myPopover>
   Hello World!
</popover-content>`

**Comment by pleerock** [comment]
fixed in 0.0.7
